test/core/contexts.json

```diff
@@ -55,12 +55,7 @@
                 "--tag", "\u523a\u8eab means sashimi",
                 "--tag", "multiple tags should be ok"
             ],
-<<<<<<< HEAD
-            "checks": ["python3-cli"],
-=======
```

Comment:
Just for my understanding, does this mean we no longer support the `python2-cli`, so customers must be on `python3`? No worries if so, but something we'd need to call out in the release notes.

Reply:
We are only supporting Python 3 to Zillow users. The feature/kfp branch is the same as what we have here: https://github.com/zillow/metaflow/blob/feature/kfp/test/core/contexts.json#L54

Python 2 has reached end of life on 1/1/2020 so I think it's ok to not support python 2.